kleva-j/ireporter-frontend-react-redux
# #164053906: Setup webpack

#### What does this PR do?
- Create the basic folder structure of the app.
- Setup Webpack for bundling app dependencies.

#### Description of the task to be completed
 - handle compilation of es6 version to es5

#### How should this be manually tested?
- Clone the repository
- Check out into the branch ft/setup-webpack/#164053906
- Install all dependencies
- run `npm start` to run the app successfully.

#### What are the relevant pivotal tracker stories?
[164053906](https://www.pivotaltracker.com/story/show/164053906)
